Quake/Makefile

```diff
@@ -70,7 +70,6 @@ define do_strip
 	$(call cmd_strip,$(1));
 endef
 endif
-CFLAGS += -std=gnu99
```

Comment:
Are you sure this is fine? There are still plenty of functions where I declare variables not at the beginning of the scope?